### PR TITLE
release/v0.12.0

### PR DESCRIPTION
**Pull request by Reagen**
Includes issues: 
#89 
#86 
#85 
#81 
#80

### CODE REVIEW

File: analysis/deep2_r23_o32/general.py [hunk 1]
```diff
@@ -34,9 +30,11 @@ def run_grid_r23_o32_analysis(dataset, n_split=3, y_correction=False,
            Version for spectrum in zoom_and_gauss_general. Default: False
     :param adaptive: bool. Set to use adaptive binning, otherwise equal
            log(R23) bins. Default: True
-    :param apply_dust: bool. Apply dust correction. Default: False
+    :param apply_dust: bool. Apply dust correction. Default: True
+                    By setting apply_dust=true, all cases are run
```
Comment (astrochun) [inline analysis/deep2_r23_o32/general.py:34]:
```suggestion
    :param apply_dust: bool. Apply dust correction. Default: True
                       By setting apply_dust=true, all cases are run
```

**Review by astrochun — CHANGES_REQUESTED** [review]
Changes look good with minor issue.  I can merge after change is implemented.

**Comment by Reagen** [comment]
Because apply_dust default remains False, I have changed the call function in the run_general_script.py to have apply_dust = True

**Comment by Reagen** [comment]
Made that change. Ready to be merged

File: analysis/deep2_r23_o32/binning/n_bins_grid_analysis.py [hunk 2]
```diff
@@ -247,11 +244,13 @@ def n_times_binned(fitspath, pdf_pages, npz_outfile, n_split, individual_ID,
 
     # Used to be called +dataset+'_binning_averages.tbl
     bin_outfile = join(fitspath, filename_dict['bin_info'])
+    print(bin_outfile)
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/n_bins_grid_analysis.py:247]:
```suggestion
```

Not needed see the log message below

File: analysis/deep2_r23_o32/binning/n_bins_grid_analysis.py [hunk 3]
```diff
@@ -247,11 +244,13 @@ def n_times_binned(fitspath, pdf_pages, npz_outfile, n_split, individual_ID,
...
     if not exists(bin_outfile):
         log.info(f"Writing: {bin_outfile}")
     else:
         log.info(f"Overwriting : {bin_outfile}")
     asc.write(tab1, bin_outfile, format='fixed_width_two_line', overwrite=True)
+    print('wrote bin_outfile')
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/n_bins_grid_analysis.py:253]:
```suggestion
```

Not needed

**Review by astrochun — CHANGES_REQUESTED** [review]
Alright two more minor suggestions

**Review by astrochun — APPROVED** [review]

**Pull request merged**